### ISSUE TITLE
[Bug] Inconsistent properties (heat capacity) of ethylene glycol / water mixture 

### ISSUE THREAD

**Issue by hobias** [DanWBR/dwsim6#538]
**Describe the bug**
Simulating a mixture of ethylene glycol and water shows wrong properties (cp, density, maybe more?) using Peng-Robinson as property package. The strange part is, that a cooler placed in between two of those streams calculates the (probably) correct amount of heat. 

So for example in DWSIM the cp of a 30/70 % mixture ethylene glycol / water is calculated to 3,37 at a temperature of 70 °C. Using the heat of an inline cooler cp would be rather 3,82.
 

**To Reproduce**

[MEG_properties.zip](https://github.com/DanWBR/dwsim/files/12522660/MEG_properties.zip)

**Expected behavior**
Display of correct properties.

**Screenshots**
![grafik](https://github.com/DanWBR/dwsim/assets/63724645/3825ca57-9a6e-4144-9676-6ef44d82bc68)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 8.6

**Comment by DanWBR** [comment]
@hobias please don't blame DWSIM for that, one should never use PR EOS to model Water + Ethylene Glycol processes.

**Comment by hobias** [comment]
I know that it is probably not the best fit, but I could not find a better solution.
And the question which remains for me is why the heat of the cooler is calculated correctly/differently? Isn't the cooler using the heat capacity?

**Comment by DanWBR** [comment]
@hobias you should be using an activity coefficient model like NRTL, UNIQUAC, UNIFAC or MODFAC. In the property package settings, you will find many options to select from to calculate the phase properties.

DWSIM uses enthalpies to calculate heat balances. How they are calculated depends on the property package and the method selected by the user.

![Captura de tela 2023-09-05 075526](https://github.com/DanWBR/dwsim/assets/2669419/2f9fa970-f64f-4408-aa34-3bd0ea049d14)
![Captura de tela 2023-09-05 075534](https://github.com/DanWBR/dwsim/assets/2669419/d8030c61-3276-4f85-9196-e46b98e09b0a)

**Comment by hobias** [comment]
Ok thanks for the clarification. I actually tried UNIFAC before and results were quite good, but not matching literture values (e.g. cp ~ 3.7 kJ/kg K instead 3.8 kJ/kg K in my case). I got the best results by using CoolProps Incompressible mixtures PP, but unfortunately it does not work in a simulation environment (probably due to #408).

Anyway thanks for your quick response and clarifications and of course the excellent work on DWSIM! I will close the issue, as it is not a bug.
